airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.tsx

```diff
@@ -0,0 +1,25 @@
+import classnames from "classnames";
+import React from "react";
+
+import styles from "./NumberBadge.module.scss";
+
+interface NumberBadgeProps {
```

Comment:
We should accept `className` as a property on all components and merge it with the other classnames for the wrapping div. Otherwise restyling, e.g. adding margin to this component in consuming places won't work.